src/opm/input/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -427,6 +427,13 @@ struct SummaryConfigContext {
         for (auto step = 0*nstep; step < nstep; ++step) {
             const auto& nodes = sched[step].network.get().node_names();
             names.insert(nodes.begin(), nodes.end());
+            // Also insert wells to be able to report network-computed THPs
+            for (const auto& node : nodes) {
+                const auto& group = sched.getGroup(node, nstep);
```

Comment:
I think this (`getGroup(node, nstep)`) will end up throwing an exception if `node` is not a group&ndash;in a run using the extended network model where there might be nodes which are not groups.  I also **suspect** that the second argument should be `step` rather than `nstep`, but I may be wrong.

It's also not completely clear to me what this loop is trying to accomplish.  Could you, perhaps, use a loop based on
[`Schedule::groupNames()`](https://github.com/OPM/opm-common/blob/aaa8af923ceeaaff60e788c224b86c77ea685498/opm/input/eclipse/Schedule/Schedule.hpp#L249-L252) and/or [`Schedule::hasGroup()`](https://github.com/OPM/opm-common/blob/aaa8af923ceeaaff60e788c224b86c77ea685498/opm/input/eclipse/Schedule/Schedule.hpp#L248) instead?

Reply:
Thanks! Didn't cater for non-group nodes, and it should indeed be step. The intention of the loop is to add names of wells belonging to the network. Could clearly loop over groupNames(), but since it is a bit arbitrary if nodes is a sub- or super-set of groups I just add a hasGroup() check.